.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  static-analysis-phpstan:
+    name: "Static Analysis with PHPStan"
+    runs-on: ubuntu-18.04
```

Comment:
ubuntu-latest

Reply:
quote for consistency

Reply:
I'm not a fan of using the "latest" tag, because it can break a build just because of some changes after "latest" becomes silently a newer version.

Reply:
cc @goetas who expressed the same opinion here: https://github.com/doctrine/sql-formatter/pull/51#discussion_r421962085

Reply:
If this is only for static analysis, it should be fine. I don't recommend it in general though.